samples/MegaDrone/src/main/cpp/MegaDroneEngine.cpp

```diff
@@ -70,23 +71,26 @@ void MegaDroneEngine::createCallback(std::vector<int> cpuIds){
     mCallback->setThreadAffinityEnabled(true);
 }
 
-void MegaDroneEngine::start(){
+bool MegaDroneEngine::start(){
     auto result = createPlaybackStream();
     if (result == Result::OK){
         // Create our synthesizer audio source using the properties of the stream
         mAudioSource = std::make_shared<Synth>(mStream->getSampleRate(), mStream->getChannelCount());
         mCallback->setSource(std::dynamic_pointer_cast<IRenderableAudio>(mAudioSource));
         mStream->start();
+        return true;
     } else {
         LOGE("Failed to create the playback stream. Error: %s", convertToText(result));
+        return false;
     }
 }
 
-void MegaDroneEngine::stop() {
+bool MegaDroneEngine::stop() {
```

Comment:
Why make this a `bool` if it always returns true and none of the callers check the return value?

Reply:
```suggestion
void MegaDroneEngine::stop() {
```